Assets/Scripts/Player.cs

```diff
@@ -3,25 +3,92 @@
 using System.Collections.Generic;
 using UnityEngine;
 
+// aight so what we want? 
+    // if MC, move according to controls
+    // if !MC: 
+        // if outside MC following position, get there quicker than movementspeed
+        // if very near position, normal follow
```

Comment:
Do we want to keep those comments?

Reply:
We wrote these for brainstorming purposes, but yeah it will be removed since it's done now.